workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseCard/index.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+
+import './index.scss';
+
+import ClayButton from '@clayui/button';
+
+import infoCircleFullIcon from '../../../../assets/icons/icon_info_circle_full.svg';
+
+const LicenseSectorCard: React.FC<any> = ({
+	icon = <img alt="Info" src={infoCircleFullIcon} />,
+	licenseDescription,
+	licensetiers,
+	lisenceType,
+}) => {
+	return (
+		<div className="license__card p-3">
+			<div className="align-items-center d-flex">
+				<span>
+					<div className="mb-1">
+						<span className="font-weight-bold">{lisenceType}</span>
+						<span className="license__card__icon ml-3">
+							{icon || infoCircleFullIcon}
+						</span>
+					</div>
+					<div>
+						<p className="license__card__text mb-0">
+							{licenseDescription}
+						</p>
+					</div>
+				</span>
+				<div className="align-items-center d-flex license__card__buttons__container p-1">
+					<span>
+						<ClayButton
+							aria-label=""
+							className="align-items-center d-flex justify-content-center license__card__buttons p-2"
+							displayType="primary"
+							onClick={() => {
+								window.location.href = origin;
+							}}
+						>
+							<ClayIcon
+								aria-label="123"
```

Comment:
Please use a descriptive aria-label

```suggestion
								aria-label="Horizontal Line"
```